lib/ui/widgets/grade_evaluation_tile.dart

```diff
@@ -0,0 +1,232 @@
+                    color: AppTheme.etsLightRed,
+                  ),
+                ),
+              ),
+              children: <Widget>[
+                evaluationsSummary(context, widget.evaluation),
+              ],
+            ),
+          ),
+        ],
+      );
+
+  Widget evaluationsSummary(BuildContext context, Evaluation evaluation) {
+    return Padding(
+      padding: const EdgeInsets.only(left: 25.0, right: 15.0),
+      child: SizedBox(
+        width: MediaQuery.of(context).size.width * 0.91,
+        child: Column(
+          children: [
+            getSummary(
+              AppIntl.of(context).grades_grade,
+              AppIntl.of(context).grades_grade_with_percentage(
+                evaluation.mark ?? 0.0,
+                evaluation.correctedEvaluationOutOf ?? 0.0,
+                getGradeInPercentage(evaluation.mark,
+                    evaluation.correctedEvaluationOutOfFormatted),
+              ),
+            ),
+            getSummary(
+              AppIntl.of(context).grades_average,
+              AppIntl.of(context).grades_grade_with_percentage(
+                evaluation.passMark ?? 0.0,
+                evaluation.correctedEvaluationOutOf ?? 0.0,
+                getGradeInPercentage(evaluation.passMark,
+                    evaluation.correctedEvaluationOutOfFormatted),
+              ),
+            ),
+            getSummary(AppIntl.of(context).grades_median,
+                validateResult(context, evaluation.median.toString())),
+            getSummary(
+                AppIntl.of(context).grades_standard_deviation,
+                validateResult(
+                    context, evaluation.standardDeviation.toString())),
+            getSummary(AppIntl.of(context).grades_percentile_rank,
+                validateResult(context, evaluation.percentileRank.toString())),
+            getSummary(AppIntl.of(context).grades_target_date,
+                getDate(evaluation.targetDate, context)),
+          ],
+        ),
+      ),
+    );
+  }
+
+  String getDate(DateTime targetDate, BuildContext context) {
+    if (targetDate != null) {
+      return DateFormat('d MMMM yyyy', AppIntl.of(context).localeName)
+          .format(targetDate);
+    }
+
+    return AppIntl.of(context).grades_not_available;
+  }
+
+  Padding getSummary(String title, String grade) {
+    return Padding(
+      padding: const EdgeInsets.only(top: 10.0, right: 15.0),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: <Widget>[
+          Text(title, style: const TextStyle(fontSize: 14)),
+          Text(grade ?? '', style: const TextStyle(fontSize: 14)),
+        ],
+      ),
+    );
+  }
+
+  double getGradeInDecimal(double grade, double maxGrade) => grade / maxGrade;
+
+  double getGradeInPercentage(double grade, double maxGrade) {
+    if (grade == null || maxGrade == null) {
+      return 0.0;
+    }
+
+    return ((grade / maxGrade) * 100).roundToDouble();
+  }
```

Comment:
seems duplicate, maybe add this in Utils or at a better place

Reply:
Good idea ! :D